node/node.go

```diff
@@ -110,20 +109,18 @@ func (n *ssvNode) Start(ctx context.Context) error {
 						zap.Uint64("committee_index", duty.GetCommitteeIndex()),
 						zap.Uint64("slot", slot))
 
-					dutyStruct := slotqueue.Duty{
-						NodeID:      validator.NodeID,
-						Duty:        duty,
-						ValidatorPK: validator.PubKey,
-						ShareSK:     validator.ShareKey,
-						Committee:   validator.Committiee,
-					}
-
-					// execute task if slot already began
-					if slot == uint64(n.getCurrentSlot()) {
+					// execute task if slot already began and not pass 1 epoch
+					currentSlot := uint64(n.getCurrentSlot())
+					if slot >= currentSlot && slot - currentSlot <= 12 {
```

Comment:
why 12? and if so why not in a param?

Reply:
to support up to 2 epochs. do you think need differ value? 
Will add flag

Reply:
please add comment and fix to 1 epoch